### DISCUSSION TITLE
Auditory 2AFC task?

### DISCUSSION THREAD

**Discussion by esi6** [l3atbc-datadog/pushkin#169] [General] [answered]
Hi,

I want to create an experiment that wasn't covered in the Jspsych tutorial over the summer. Mainly, an auditory 2AFC task, where the participant hears one word, followed by a brief pause, followed by another word. They then provide a response about which one was the target item.

I tried adapting the tutorials to my purposes, but I ran into an issue with the responses. In the tutorial, the responses (such as in the lexical decision task) appear to be locked to a visual stimulus. I tried utilizing this setup by simply replacing the visual stimulus with an auditory one, but because my audio files are so short (~2sec), the responses aren't getting logged and the experiment is unable to progress beyond the first test trial.

Is there a way to lock responses to a test slide, like in psychopy or e-prime? Or another way to get around this issue? I do not necessarily want participants responses to be locked to the second item, as I want to give them sufficient time to respond.

Thanks!

**Comment by parkerrobbins** [comment, marked as answer]
Hi, @esi6!

This is a great question but is better suited to the [jsPsych discussion board](https://github.com/jspsych/jsPsych/discussions). 

Please let me know if you have any questions!

**Comment by restmegan** [comment]
Hi @esi6 ! I just wanted to check in to see how Pushkin has been going for you. The Pushkin team is online today answering questions so if there's anything you need help with please let us know!
